### PR TITLE
rubocop: default hash syntax.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
As-of 3.1: this mean that you omit the hash value if the name is the same as the key.

We're allowing this already and it didn't make sense to land until the bulk of the other RuboCop 3.1 changes did but, now we're ready, it is more concise and a pattern that people will need to understand anyway.

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]
That's some proper magic right there

**Comment by MikeMcQuaid** [comment]
`Error: You called sig twice without declaring a method in between`

any idea what's going on or how to debug this @dduugg?

File: Library/Homebrew/cask/artifact/artifact.rb [hunk 1]
```diff
@@ -33,9 +33,6 @@ def resolve_target(target)
       end
 
       sig { params(cask: Cask, source: T.any(String, Pathname), target: T.any(String, Pathname)).void }
-      def initialize(cask, source, target:)
-        super(cask, source, target: target)
-      end
```
Comment (Bo98) [inline Library/Homebrew/cask/artifact/artifact.rb:38]: Sig needs removing here too

Comment (reitermarkus) [inline Library/Homebrew/cask/artifact/artifact.rb:38]: Probably this.

Comment (reitermarkus) [reply Library/Homebrew/cask/artifact/artifact.rb:38]: As long as `super` has the same signature, it should be fine.

Comment (Bo98) [reply Library/Homebrew/cask/artifact/artifact.rb:38]: yeah misread the super thing initially - sig should still be removed though

**Comment by MikeMcQuaid** [comment]
Thanks all for the catch 👁️

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**